lib/base-command.js

```diff
@@ -72,5 +73,13 @@ class BaseCommand {
       { code: 'ENOWORKSPACES' }
     )
   }
+
+  async setWorkspaces (filters) {
+    // TODO npm guards workspaces/global mode so we should use this.npm.prefix?
```

Comment:
I prefer using `this.npm.localPrefix` to avoid the extra jump when reading across those

Reply:
I do not like that we have both. It is prone to error.